[Rust] `widestring` don't contain `U16CString`

```
error[E0432]: unresolved import `widestring::U16CString`
   --> libparsec\crates\platform_http_proxy\src\native\windows.rs:5:27
    |
5   | use widestring::{U16CStr, U16CString};
    |                           ^^^^^^^^^^
    |                           |
    |                           no `U16CString` in the root
    |                           help: a similar name exists in the module: `U16CStr`
    |
note: found an item that was configured out
   --> C:\Users\runneradmin\.cargo\registry\src\index.crates.io-6f17d22bba15001f\widestring-1.0.2\src\lib.rs:260:20
    |
260 | pub use ucstring::{U16CString, U32CString, WideCString};
    |                    ^^^^^^^^^^
    = note: the item is gated behind the `alloc` feature

error[E0412]: cannot find type `U16CString` in crate `widestring`
    --> libparsec\crates\platform_http_proxy\src\lib.rs:20:33
     |
20   |       pac_url: Option<widestring::U16CString>,
     |                                   ^^^^^^^^^^ help: a struct with a similar name exists: `U16CStr`
     |
    ::: C:\Users\runneradmin\.cargo\registry\src\index.crates.io-6f17d22bba15001f\widestring-1.0.2\src\ucstr.rs:938:1
     |
938  | / ucstr_common_impl! {
939  | |     /// C-style 16-bit wide string slice for [`U16CString`][crate::U16CString].
940  | |     ///
941  | |     /// [`U16CStr`] is to [`U16CString`][crate::U16CString] as [`CStr`][std::ffi::CStr] is to
...    |
1106 | |     fn display() -> {}
1107 | | }
     | |_- similarly named struct `U16CStr` defined here
     |
note: found an item that was configured out
    --> C:\Users\runneradmin\.cargo\registry\src\index.crates.io-6f17d22bba15001f\widestring-1.0.2\src\lib.rs:260:20
     |
260  | pub use ucstring::{U16CString, U32CString, WideCString};
     |                    ^^^^^^^^^^
```

https://github.com/Scille/parsec-cloud/actions/runs/7877819199/job/21494769371?pr=6402

--- a/libparsec/crates/platform_http_proxy/Cargo.toml
+++ b/libparsec/crates/platform_http_proxy/Cargo.toml
@@ -18,4 +18,4 @@ reqwest = { workspace = true }
 
 [target.'cfg(target_os = "windows")'.dependencies]
 windows-sys = { workspace = true, features = ["Win32_Networking_WinHttp", "Win32_Foundation"] }
-widestring = { workspace = true }
+widestring = { workspace = true, features = ["alloc"] }

